### ISSUE TITLE
Game needs to be more "Sit down and Play" - UX/UI

### ISSUE THREAD

**Issue by snowschu** [josefnpat/roguecraft-squadron#617]
_This is mostly for demoing but can be applied to the game in general._

Right now, the game must have a babysitter. To help set the game up, to explain the game, and to be available for questions. One cannot leave the game alone and hope people figure out how to 'begin' the game.

Sure, after we've explained it the first time, the players kind of get it. But the point is the first impression is very "deer in headlights". A little scary and confusing.

I would like this to be discussion on how we can improve this for demoing and, ultimately, making it easy for players to just pick it up and start a game.

**TLDR**: Less babysitting and teaching from the booth watcher. More allowance for player discovery, booth watcher being available for questions from the player and observers, and ability for more guerrilla demo set ups.

**Comment by softsound** [comment]
Actionable steps:
1) Better tutorial
2) Have win/lose restart to lobby
3) Have control reference/overlay easy to find. 

4)Set up screen recording on obs?

I don't know what else can be done.

**Comment by snowschu** [comment]
Please include examples or breakdowns of how these actions would take place, from your perspective. I will state that these are not actions yet.

**Comment by snowschu** [comment]
1. a) A "better tutorial" isn't actionable. Write an example or two of a better tutorial (UX steps).
b) The better tutorial would also have to extend to the lobby as it is not clear what any of the clickable elements actually mean for the players. (To new players).

2. This would be nice. Ideally it would go "Victory/Defeat" -> Thanks for playing spiel -> Lobby

3. In game access to this would help. Still waiting on `Options` to get added to the game menu.

4. This isn't useful for idea of `sit down and play` as it doesn't address the player's needs. Recording the players was to help with testing and possibly use for promotional material. I have brought up recording for bug catching/game crashes as well. The fact is that no-one (including myself) has truly reviewed this footage in a timely manner.

**Comment by josefnpat** [comment]
Overall, I agree with this ticket. some notes:

1. A better tutorial can be produced by having folks sit down, and not guiding them. Observe and document what they do, and see if they can figure it out. If they can't, asking them questions is the best way to get good feedback:
  a. What are you trying to do?
  b. (followup) How would you do it?
2. Current UX workflow is: Victory/Defeat -> Specacte/Quit -> Main menu.
3. #373
